Content.Server/Chat/ChatManager.cs

```diff
@@ -217,5 +228,10 @@ private void _onMaxLengthRequest(ChatMaxMsgLengthMessage msg)
             response.MaxMessageLength = MaxMessageLength;
             _netManager.ServerSendMessage(response, msg.MsgChannel);
         }
+
+        public void RegisterChatTransform(Func<IEntity, string, string> handler)
```

Comment:
This would be handled better by declaring a delegate in IChatManager, like `
        delegate string TransformChat(IEntity speaker, string message);`, which then allows you to turn this into `public void RegisterChatTransform(TransformChat handler)`.